pkgs/development/python-modules/python-nest/default.nix

```diff
@@ -0,0 +1,28 @@
+{ buildPythonPackage, fetchFromGitHub, lib, python, python-dateutil, requests
+, six, sseclient-py }:
+
+buildPythonPackage rec {
+  pname = "python-nest";
+  version = "4.1.0";
+
+  # pypi does not contain tests, using github sources instead
```

Comment:
This comment does not really make sense when there are no tests at all. In which case we can fetch this package from pypi.